usr/lib/systemd/system/cernvm-contextualization.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=CernVM cluster online API contextualization
+After=cloud-init.service
+Before=cloud-config.service multi-user.target
+Wants=cloud-init.service cloud-config.service
```

Comment:
I think we should add `ConditionPathExists=/mnt/.rw/context/ucontext`.  I think it should also be called cernvm-cluster-contextualization.service